web/tests/user_search.test.cjs

```diff
@@ -118,7 +124,8 @@ test("clear_search", ({override}) => {
 });
 
 test("clear_search", ({override}) => {
-    override(realm, "realm_presence_disabled", true);
+    realm.realm_presence_disabled = true;
```

Comment:
I ended up doing `realm.realm_presence_disabled` because the override wasn't working in my new test (it would show up as undefined even though I was overriding it as true) and it worked this way instead. I'm still not completely sure why the other thing was broken, so let me know if a different solution would be better.